Formula/kahip.rb

```diff
@@ -1,9 +1,8 @@
 class Kahip < Formula
   desc "Karlsruhe High Quality Partitioning"
   homepage "https://algo2.iti.kit.edu/documents/kahip/index.html"
-  url "https://algo2.iti.kit.edu/schulz/software_releases/KaHIP_2.12.tar.gz"
-  sha256 "b91abdbf9420e2691ed73cea999630e38dfaf0e03157c7a690a998564c652aac"
-  revision 2
+  url "https://github.com/KaHIP/KaHIP/archive/v3.10.tar.gz"
```

Comment:
This is the tarball linked to in the [homepage](https://algo2.iti.kit.edu/documents/kahip/index.html).

> - [KaHIP_3.10.tar.gz](https://github.com/KaHIP/KaHIP/archive/v3.10.tar.gz) (or available on GitHub [here](https://github.com/KaHIP/))